### PR TITLE
Add GH token to lookml-generator

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

**Comment by fbertsch** [comment]
@jasonthomas this token needs `repo` access (full control of all private repos), and `read:org`.

File: dags/probe_scraper.py [hunk 1]
```diff
@@ -121,6 +121,7 @@
             "LOOKER_INSTANCE_URI": "https://mozilla.cloud.looker.com",
             "LOOKER_API_CLIENT_ID": Variable.get("looker_api_client_id_prod"),
             "LOOKER_API_CLIENT_SECRET": Variable.get("looker_api_client_secret_prod"),
+            "GITHUB_ACCESS_TOKEN": Variable.get("github_access_token"),
```
Comment (jasonthomas) [inline dags/probe_scraper.py:124]:
```suggestion
            "GITHUB_ACCESS_TOKEN": Variable.get("dataops_looker_github_secret_access_token"),
```

File: bin/run [hunk 2]
```diff
@@ -145,6 +145,7 @@ init_variables() {
     airflow variables -s "looker_api_client_secret_staging" "looker_api_client_secret_staging"
     airflow variables -s "looker_api_client_id_prod" "looker_api_client_id_prod"
     airflow variables -s "looker_api_client_secret_prod" "looker_api_client_secret_prod"
+    airflow variables -s "github_access_token" "github_access_token"
```
Comment (jasonthomas) [inline bin/run:148]:
```suggestion
    airflow variables -s "github_access_token" "dataops_looker_github_secret_access_token"
```

**Review by jasonthomas — APPROVED** [review]

**Comment by jasonthomas** [comment]
Added token as `dataops_looker_github_secret_access_token` on WTMO

File: bin/run (hunk 2 above)
Comment (fbertsch) [inline bin/run:148]:
```suggestion
    airflow variables -s "dataops_looker_github_secret_access_token" "dataops_looker_github_secret_access_token"
```

**Pull request merged**